packages/lib/src/validators/interface.ts

```diff
@@ -10,7 +10,7 @@ export class InterfaceVisitor extends GirVisitor {
 
             if (!GObject) {
                 throw new Error(
-                    `GObject.Object could not be found while generating ${node.namespace.name}.${node.name}`
+                    `GObject.Object could not be found while generating ${node.namespace.namespace}.${node.name}`
```

Comment:
I think the original was correct :thinking:

Reply:
I'll have another look, I think I had renamed the proboerty

Reply:
Sorry for the slow turn around, I find this renaming a bit confusing. `namespace.name` matches the pattern elsewhere.

Reply:
I see this is aligning the previous properties in `GirModule` which calls the prop `namespace`, I still find it confusing but I don't think this is a blocking issue.